source/server/BUILD

```diff
@@ -352,23 +288,32 @@ envoy_cc_library(
         ":lds_api_lib",
         ":transport_socket_config_lib",
         ":well_known_names_lib",
+        "//include/envoy/network:connection_interface",
         "//include/envoy/server:active_udp_listener_config_interface",
+        "//include/envoy/server:api_listener_interface",
         "//include/envoy/server:filter_config_interface",
         "//include/envoy/server:listener_manager_interface",
         "//include/envoy/server:transport_socket_config_interface",
         "//include/envoy/server:worker_interface",
+        "//source/common/api:os_sys_calls_lib",
+        "//source/common/common:empty_string",
         "//source/common/config:utility_lib",
         "//source/common/config:version_converter_lib",
         "//source/common/http:conn_manager_lib",
         "//source/common/init:manager_lib",
+        "//source/common/network:connection_balancer_lib",
         "//source/common/network:listen_socket_lib",
+        "//source/common/network:resolver_lib",
         "//source/common/network:socket_option_factory_lib",
         "//source/common/network:utility_lib",
         "//source/common/protobuf:utility_lib",
+        "//source/common/stream_info:stream_info_lib",
         "//source/extensions/filters/listener:well_known_names",
         "//source/extensions/filters/network/http_connection_manager:config",
         "//source/extensions/transport_sockets:well_known_names",
         "@envoy_api//envoy/admin/v3:pkg_cc_proto",
+        "@envoy_api//envoy/api/v2/listener:pkg_cc_proto",
+        "@envoy_api//envoy/api/v2:pkg_cc_proto",
```

Comment:
v2:pkg... sorts before v2/listen... (colon delimited first.) Switch these 2 to get linux-format to pass.

Reply:
Thanks - done.